.github/workflows/django.yml

```
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8,3.9,'3.10', '3.11']
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
```

Break into one-per-line to reduce future diff noise:

```suggestion
        python-version:
        - '3.8'
        - '3.9'
        - '3.10'
        - '3.11'
        - '3.12'
```